timescaledb/how-to-guides/uninstall/uninstall-timescaledb.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Uninstall TimescaleDB
+excerpt: Uninstall TimescaleDB on macOS
+keywords: [uninstall]
+---
+
+# Uninstall TimescaleDB
+
+If you installed TimescaleDB with Homebrew or MacPorts, you can uninstall it
+without having to uninstall PostgreSQL.
```

Comment:
Perhaps we can go into a little more detail here, talk about how it's an extension, so you can drop the extension and keep PostgreSQL.